css/styles.css

```diff
@@ -0,0 +1,234 @@
+/* pasek nawigacji------------ */
+nav .logo {
+    font-family: 'Pacifico', cursive;
+    font-size: xx-large;
+    margin-left: 8%;
+flex-basis: 40%;
+}
+nav .menu a{
+    font-family: 'Varela Round', sans-serif;   
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: bold;    
+}
+nav{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;    
+}
+
+/* pierwsza stronka-------------- */
+.okno{
+    width: 100vw;
+    height: 100vh;
+    background-image: url(/photo/splash-milk-strawberry.jpg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin: 0;
+}
+
+.textokno{
+    width: 30%;
+    height: 10%;
+    position: relative;
+    left: 10%;
+    top: 10%;
+
+}
+.getfree button{height:50px;width: 250px; background-color: rgb(255, 115, 0); font-size: x-large; box-shadow: black 2px 2px 5px; border-width: 0px; color: rgb(243, 236, 236);font-weight: 100; font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif; text-transform: uppercase; position: relative; left: 10%; top: 150px;}
```

Comment:
Don't format it like that. each style should be in separate line. The readability is terrible